### PR TITLE
[DB] Slight refactor of DB things

### PR DESCRIPTION
**Pull request by avano**
//test: `@rest and @database`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java [hunk 1]
```diff
@@ -78,42 +84,21 @@ public int executeSQLGetUpdateNumber(String sqlCommnad) {
...
-            while (resultSet.next()) {
-                records = resultSet.getInt(1);
-            }
-        } catch (SQLException ex) {
-            log.error("Error: " + ex);
-        }
-        log.debug("Number of records: " + records);
-
-        return records;
+        String sql = "SELECT * FROM " + tableName.toUpperCase() + (args.length == 2 ? " WHERE " + args[0] + " LIKE '" + args[1] + "'" : "");
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java]: While I do admit this is a bit clearer than the original, I'd say we should probably have two overloaded versions of this method.

File: utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java [hunk 2]
```diff
@@ -175,21 +172,21 @@ public void verifyIntegrationWithTask(String task) {
         assertThat(newTaskCount).isGreaterThan(oldTaskCount + 5);
     }
 
-    @And("^.*checks? that query \"([^\"]*)\" has \"(\\w+)\" output$")
-    public void checkNumberValuesExistInTable(String query, int number) {
+    @Then("^.*checks? that query \"([^\"]*)\" has \"(\\w+)\" output$")
+    public void checkNumberValuesExistInTable(String query, Integer number) {
```
[the remark this answers is not in the document]
Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java]: :thinking: 

it was originally there and mcada changed that in his PR and during the rebase his change got reverted :yum: I'll change it back

File: utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java (hunk 1 above)
Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/utils/DbUtils.java]: :+1:

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**